app/views/content/is-teaching-right-for-me/maths.md

```diff
@@ -2,6 +2,10 @@
 title: Become a maths teacher
```

Comment:
The header/article/promo for maths hasn't been moved properly which is why the nav card appears on the ITRFM page but you get a 404 if you click on the card

![image](https://github.com/DFE-Digital/get-into-teaching-app/assets/87642394/52928cdf-6519-4b6e-a401-b5d1d342b812)